component/common/loki/client/client.go

```diff
@@ -363,6 +363,8 @@ func (c *client) sendBatch(tenantID string, batch *batch) {
 				}
 			}
 			return
+		} else {
+			level.Error(c.logger).Log("err", err)
```

Comment:
It does not, I was trying to catch the case where the status check causes it to break before logging, but it will be caught by the final error check so removing.